todo_app/data/IndexViewModel.py

```diff
@@ -0,0 +1,69 @@
+from todo_app.data.Item import Item
+from todo_app.data.trello_items import get_trello_lists
+
+class ViewModel:
+    def __init__(self, items, lists, errored):
+        self._items = items
+        self._errored = errored
+        self._sorted_items = {}
+        self._lists = lists
+        self.sort_items()
+
+    def sort_items(self):
+        trello_lists = self.trello_lists
+
+        for trello_list in trello_lists:
+            list_name = trello_list["name"]
+            self._sorted_items[list_name] = []
+
+        #sort items into correct list
+        for item in self._items:
+            self._sorted_items[item.status].append(item)
+
+    @property
+    def sorted_items(self):
```

Comment:
Depending on our data, splitting out the sort_items and sorted_items functions can be important for efficiency to save us sorting repeatedly, but here I think it's reasonable to assume we're unlikely to hit more than a few hundred to-dos that need sorting, so it might make more sense to have this property actually re-create the sorted list each time it's called, rather than relying on users always remembering to explicitly call `sort_items` elsewhere